ls1intum/Artemis
# Style bamboo logs correct.

### Checklist

### Motivation and Context
This fixes https://github.com/ls1intum/Artemis/issues/1788. The build logs are now displayed correctly (again).

### Description
The object returned from Bamboo has 2 attributes per LogEntry: `log` and `unstyledLog`.
`log` contains escaped HTML characters, so we use `unstyledLog`.

### Steps for Testing
1. Check https://github.com/ls1intum/Artemis/issues/1788 to understand the context
2. Create a build failure (preferably using a Haskell exercise)
3. Verify that the build logs does not contain any escaped characters

### Screenshots
![image](https://user-images.githubusercontent.com/5084100/86269926-bd2f4100-bbca-11ea-858a-6caf169d823f.png)
